src/components/Book.jsx

```diff
@@ -3,22 +3,54 @@ import { useDispatch } from 'react-redux';
 import { removeAPI } from '../redux/books/books';
```

Comment:
- [ ] As the previous reviewer mentioned your page has too much margin on the right, kindly consider removing it so the page is identical to the zipline design and uses white space more effectively 

![Screenshot_20220819_194140](https://user-images.githubusercontent.com/25227022/185685911-3d9af59a-a462-4b66-9916-528584707288.png)

Reply:
@Ismailco The previous reviewer emphazed on horizontal scroll and not what you are pointing out. He only marked that section to show me that could be the possible cause. On my screen, there is no such issue with spacing. I don't think this is an issue that demands for another fix. 

Find below how it looks like in my screen
![image](https://user-images.githubusercontent.com/30408871/185687990-a6d20130-d837-4906-8050-937fa4fcab5e.png)

Reply:
@Ismailco I just added to borders so I can see it clearly. What exactly is it that you want to me to do. I don't understand you quite well. I need you to clarify on this please

Reply:
![image](https://user-images.githubusercontent.com/30408871/185689794-12279489-74d3-4f11-ae5f-b0985ee33552.png)
That's the design above. What's the significant difference please?

Reply:
@ogagaoghene, There is a margin on the right of your book cards, and the preview reviewer mentioned this

![Screenshot_20220819_203829](https://user-images.githubusercontent.com/25227022/185694790-24924a03-4db3-4a03-9f5b-13d237c15fff.png)

Reply:
I have made some adjustments as requested. This is my last review, I ask for a positive review so I can work on other pending tasks.